packages/python/plotly/plotly/tests/test_optional/test_offline/test_offline.py

```diff
@@ -89,7 +89,6 @@ def test_default_mpl_plot_generates_expected_html(self):
             # just make sure a few of the parts are in here
             # like PlotlyOfflineTestCase(TestCase) in test_core
             self.assertTrue(data_json in html)  # data is in there
-            self.assertTrue(layout_json in html)  # layout is in there too
```

Comment:
why does this line need to go?

Reply:
Oh, I should have mentioned that.  Without sorting, there is a property order difference in the layout string here.  I spent a half hour trying to track it down but didn't get all the way there, it looked like it was related to some combination of fig.to_dict() and later figure validation not always preserving the property order.  All of the properties are there, they just get reordered in the HTML output compared to the plain json serialization.

Reply:
hmmm ok.